modules/eventlist.php

```diff
@@ -24,111 +24,29 @@
  *  $Id$
  */
 
-function GetEventList($year=NULL, $month=NULL, $day=NULL, $forward=0, $customerid=0, $userid=0, $type = 0, $privacy = 0, $closed = '') {
```

Comment:
Zmieniasz prototyp funkcji. Wcześniej $closed mogło mieć 3 stany:
0 - otwarte,
1 - zamknięte,
'' - dowolne,
Jak to teraz będzie rozróżniane?

Reply:
0 otwarte, 1 zamknięte, null - dowolne